internal/rps/message.go

```diff
@@ -165,11 +165,8 @@ func (p Payload) CreateMessageRequest(flags flags.Flags) (Message, error) {
 	if payload.CurrentMode != 0 {
 		if flags.Password == "" {
 			for flags.Password == "" {
-				fmt.Println("Please enter AMT Password: ")
-				// Taking input from user
-				_, err = fmt.Scanln(&flags.Password)
-				if err != nil {
-					return message, err
+				if _, err := flags.ReadPasswordFromUser(); err != nil {
+					return message, errors.New("Missing or incorrect password")
```

Comment:
there should be an existing error for this in the utils package

Reply:
Done